.github/workflows/main.yml

```diff
@@ -69,29 +71,67 @@ jobs:
         with:
           chart-dir: ${{ env.CHART_DIR }}
 
-      - name: Get commit message
-        id: get-commit-message
-        shell: bash
-        run: |
-          MSG=$(git log --pretty=format:%s --max-count=1)
-          echo "message=$MSG" >> $GITHUB_OUTPUT
+  release:
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.event_name == 'push'
```

Comment:
Please remove this check so that we can check most of this logic in PRs

Reply:
fixed